cura/CrashHandler.py

```diff
@@ -451,6 +451,9 @@ def _sendCrashReport(self):
         os._exit(1)
 
     def show(self):
+        Application.getInstance().getPreferences().setValue("general/last_shutdown_severity",
+                                                            Application.ShutdownSeverity.CrashHandler)
```

Comment:
This won't work if Cura crashes within 10 seconds, which is the delay for saving the preferences file. But maybe that's acceptable.

Reply:
Sorry, wrong part of code.